specs/lib/gltfStatisticsSpec.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+var assert = require("assert");
+var fs = require('fs');
+var addDefaults = require('../../lib/addDefaults');
+var addPipelineExtras = require('../../lib/addPipelineExtras');
+var getStatistics = require('../../lib/gltfStatistics').getStatistics;
+var readGltf = require('../../lib/readGltf');
+
+describe('gltfStatistics', function() {
+
+    it('should return stats', function(){
+        readGltf('specs/data/statistics/Cesium_Air.gltf')
+            .then(function(gltf) {
+                var stats = getStatistics(gltf);
+
+                assert.equal(stats.buffersSizeInBytes, 172256);
+                assert.equal(stats.numberOfImages, 2);
+                assert.equal(stats.numberOfExternalRequests, 0);
+
+                assert.equal(stats.numberOfDrawCalls, 3);
+                assert.equal(stats.numberOfRenderedPrimitives, 5840);
+
+                assert.equal(stats.numberOfNodes, 7);
+                assert.equal(stats.numberOfMeshes, 2);
+                assert.equal(stats.numberOfMaterials, 2);
+                assert.equal(stats.numberOfAnimations, 2);
+            });
+    });
```

Comment:
I left a similar comment in the `3d-tiles-tools` pr - make sure to check that the promise resolves, like in `writeImagesSpec`.

```javascript
    it('writes an embedded buffer', function(done) {
        var gltf = clone(testGltf);
        var options = {
            outputPath : outputPath,
            embed : true,
            embedImage : true,
            createDirectory : true
        };
        
        expect(writeGltf(gltf, options)
            .then(function() {
                expect(gltf.images.Cesium_Logo_Flat_Low.extras).not.toBeDefined();
                expect(gltf.images.Cesium_Logo_Flat_Low.uri).toEqual(imageUri);
            }), done).toResolve();
    });
```